src/vendor/wordpress/demo/contact-form/success.twig

```diff
@@ -0,0 +1,62 @@
+      <div class="contact-form-submission">
+        <p class="go-back-message"> <a class="link" href="/testing-contact-form/?contact-form-hash=test">Go back</a> </p>
+        <h4 id="contact-form-success-header">Your message has been sent</h4>
+
+        <p></p>
+        <div class="field-name">Name:</div>
+        <div class="field-value">Test McTest</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Email:</div>
+        <div class="field-value">tyler@cloudfour.com</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Who is your favorite manager?:</div>
+        <div class="field-value">Some other guy, I can’t quite remember</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Which are you hungry for?:</div>
+        <div class="field-value">Pizza, Sandwiches, Chocolate</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Why are you writing today?:</div>
+        <div class="field-value">Sorry, my cat sat on my keyboard</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Text:</div>
+        <div class="field-value">This is text</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Date:</div>
+        <div class="field-value">Today</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Phone:</div>
+        <div class="field-value">5555555555</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Website:</div>
+        <div class="field-value">https://tylersticka.com</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Message:</div>
+        <div class="field-value">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse condimentum ultricies mi at iaculis. Aenean quis accumsan est. Duis auctor luctus dolor quis imperdiet. Duis efficitur molestie purus, a ullamcorper nulla egestas sed. Proin ut tortor ex. Nam consequat lectus feugiat, feugiat erat at, fringilla tortor.</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Example of a checkbox:</div>
+        <div class="field-value">Yes</div>
+        <p></p>
+        <p></p>
+        <div class="field-name">Consent:</div>
+        <div class="field-value">Yes</div>
+        <p></p>
```

Comment:
It's wild that this is the markup Jetpack forms output. This has got to be used on so many websites... (Not a blocker at all, just... come on Jetpack...)